src/Microsoft.Framework.PackageManager/Publish/PublishManager.cs

```diff
@@ -141,12 +145,8 @@ public bool Publish()
                     if (string.IsNullOrEmpty(runtimeHome))
                     {
                         var runtimeGlobalPath = Environment.GetEnvironmentVariable(EnvironmentNames.GlobalPath);
```

Comment:
This is an environment variable `DNX_GLOBAL_PATH`, looks like some global DNX home set by MSI installer. Will make this a DnuFolderPath.